run.py

```diff
@@ -799,36 +799,63 @@ def speaker_info(speaker_uuid: str, core_version: Optional[str] = None):
     @app.get(
         "/downloadable_libraries",
         response_model=List[DownloadableLibrary],
-        tags=["その他"],
+        tags=["ライブラリ管理"],
     )
     def downloadable_libraries():
         """
-        ダウンロード可能なモデル情報を返します。
+        ダウンロード可能なライブラリの情報を返します。
 
         Returns
         -------
         ret_data: List[DownloadableLibrary]
         """
-        try:
-            manifest = engine_manifest_loader.load_manifest()
-            # APIからダウンロード可能な音声ライブラリを取得する場合
-            if manifest.downloadable_libraries_url:
-                response = requests.get(manifest.downloadable_libraries_url, timeout=60)
-                ret_data: List[DownloadableLibrary] = [
-                    DownloadableLibrary(**d) for d in response.json()
-                ]
-            # ローカルのファイルからダウンロード可能な音声ライブラリを取得する場合
-            elif manifest.downloadable_libraries_path:
-                with open(manifest.downloadable_libraries_path) as f:
-                    ret_data: List[DownloadableLibrary] = [
-                        DownloadableLibrary(**d) for d in json.load(f)
-                    ]
-            else:
-                raise Exception
-        except Exception:
-            traceback.print_exc()
-            raise HTTPException(status_code=422, detail="ダウンロード可能な音声ライブラリの取得に失敗しました。")
-        return ret_data
+        manifest = engine_manifest_loader.load_manifest()
+        if not manifest.supported_features.manage_library:
+            raise HTTPException(status_code=404, detail="この機能は実装されていません")
+        return library_manager.downloadable_libraries()
```

Comment:
最初に機能判定しておくとloadで失敗する可能性を考えなくて済むかも
```suggestion
        if not manifest.supported_features.manage_library:
            raise HTTPException(status_code=404, detail="この機能は実装されていません")
        manifest = engine_manifest_loader.load_manifest()
        return library_manager.downloadable_libraries()
```

Reply:
あっ！！　これダメですね、ボケてました。。。